CONTRIBUTORS

```diff
@@ -61,4 +62,4 @@ Collabora
  Ricardo Cañuelo
 Dipanjan Das
 Daimeng Wang
-Jukka Kaartinen
+Jukka Kaartinen
```

Comment:
Again, please don't change the whitespace. (missing newline)

Reply:
I think it should be good now